spec/requests/api/docs/v3/applications_spec.rb

```diff
@@ -21,12 +33,6 @@
     let(:resource) { application }
   end
 
-  it_behaves_like "an API accept application endpoint documentation",
```

Comment:
Why can't we reuse this for V3? Seems like the only difference is the params and it can be override I guess:

Something like:
```
  it_behaves_like "an API accept application endpoint documentation",
                  "/api/v3/npq-applications/{id}/accept",
                  "#/components/schemas/ApplicationResponse" do
    let(:resource) { application }
    let(:params) do
      {
        "data": {
          "type": "npq-application-accept",
          "attributes": {
            funded_place: true,
            schedule_identifier: "npq-leadership-spring",
          },
        },
      }
    end
  end
```